### ISSUE TITLE
Library no longer normalizes its input

### ISSUE THREAD

**Issue by wcauchois** [super-ienien/percent-round#4]
Hi! Thanks for making this library.

I noticed a change in behavior between 1.x and 2.x that tripped me up. It seems that percent-round no longer normalizes its input:

**Before (1.0.3)**

```
> percentRound([1,2,33333])
[ 0, 0, 100 ]
```

**After (2.1.1)**

```
> percentRound([1,2,33333])
[ -11078, -11077, 22255 ]
```

I guess its expected that you always input percentages now, instead of numbers that will be automatically normalized? However, I think the normalization was convenient. Also, assuming this function expects input that sums to 100... that can actually be hard in JavaScript when there are floating-point-precision blips. It'd be nice if this library was guaranteed to handle the normalization for you.

**Comment by super-ienien** [comment]
Ok,

I will fix this as soon as possible

**Comment by super-ienien** [comment]
fixed in 2.2.0
